chromium/browser_type_test.go

```diff
@@ -32,6 +12,7 @@ import (
 	"github.com/grafana/xk6-browser/common"
 )
 
+//nolint:funlen
 func TestBrowserTypeFlags(t *testing.T) {
```

Comment:
Yes, it happens every time:

<img width="774" alt="Screen Shot 2022-03-08 at 13 55 46" src="https://user-images.githubusercontent.com/621232/157224929-d87fe70e-ef7f-45f7-b1b8-d465c3b840b9.png">

Makes sense 👍

Reply:
See: https://github.com/grafana/xk6-browser/runs/5463300546?check_suite_focus=true

Reply:
Hhmm I'm really not sure why it's never triggered for me on tests then, in _any_ Go project I've worked on (k6 particularly has a bunch of long tests)... :confused: This `golangci-lint` flakiness is more than annoying.

In any case, it would make more sense to just disable it globally for tests, since test functions are usually more than 60 lines long, and having this `nolint` on almost every test would be annoying. Please disable it in the config file [as suggested here](https://github.com/ultraware/funlen/issues/5#issuecomment-579683174), or somehow in your editor.